src/cljs/main/broadfcui/common/components.cljs

```diff
@@ -796,9 +796,9 @@
        (if (:label props)
          [:span {}
           (:label props) " "
-          (style/create-link
-           {:text (icons/icon {} (if (:collapsed? @state) :expand :collapse))
-            :onClick #(swap! state assoc :collapsed? (not (:collapsed? @state)))})
+          (links/create-internal
+           {:onClick #(swap! state assoc :collapsed? (not (:collapsed? @state)))}
```

Comment:
`update :collapsed? not`?
(I know these aren't yours, but as long as we're touching them, right?)